### PR TITLE
add-CTA-on-every-observability-integration-page

### LINKED ISSUE #5234: Add CTA to every integration falling under the observability category.
### Current Behavior

Each [Meshery Integrations](https://layer5.io/cloud-native-management/meshery/integrations)' page which comes under observability, needs to have a CTA to [Performance page.
](https://layer5.io/projects/cloud-native-performance)
### Desired Situation

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR fixes #5234 
Take a look: https://65955faffa5ed705d1f6d288--layer5.netlify.app/cloud-native-management/meshery/integrations/aws-cloudtrail

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a6bd6feac1cbca834bcbc10b7762954e5d58524c at: https://659290d17632d6a874eb6f57--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
No: Know More

Yes: Learn More

Please change the button text.

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/cta-performance.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <Container>
+        <div className="CTAbody">
+          <div>
+            <img className="logo" src={SMPLogo} />
+          </div>
+          <div className="text">
+            <h2><span>Cloud Native Performance</span></h2>
+            <h3 className="subtext">Standardizing Service Mesh Value Measurement</h3>
+            <Button primary url="/projects/cloud-native-performance" className="button" external={false}>
+                            Know more
```
Comment (vishalvivekm) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/cta-performance.js:86]:
```suggestion
                            Learn More
```

**Comment by vishalvivekm** [comment]
> No: Know More
> 
> Yes: Learn More
> 
> Please change the button text.

Done

**Comment by l5io** [comment]
🚀 Preview for commit 08fdf6f7aef85df7fb15d4954acf9885d4b451d2 at: https://65955faffa5ed705d1f6d288--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 24772b0546a119081991f069282435e6404b1dde at: https://6606a2551ddf01988fc4b861--layer5.netlify.app

**Pull request merged**